boot.php

```diff
@@ -1083,7 +1092,7 @@ function remove_baseurl($orig_url){
 	function register_template_engine($class, $name = '') {
 		if ($name===""){
```

Comment:
Standards: Can you please wrap operators with spaces and add a space before the brace?